public/css/showoff.css

```diff
@@ -72,6 +72,15 @@
   }
 }
 
+.content {
+  position: relative;
+  /* for some reason, setting top to 50% is pushing the content too far down, no idea why */
+  top: 42%;
```

Comment:
Nah, don't worry about it for now. We can put in a conditionally loaded stylesheet with ie8 corrections later if we decide we care.